test/clj/rems/api/testing.clj

```diff
@@ -161,6 +161,11 @@
   (assert (:success body) (pr-str body))
   body)
 
+(defn assert-unauthorized [response]
+  (response-is-unauthorized? response)
+  {:status (:status response)
+   :body (read-body response)})
+
```

Comment:
This is named `assert-` yet it does no such thing wrt. to unauthorized. Copying some fields from the response makes no sense to me.

Reply:
hmm, I was referencing the logic of the other functions in the file, but I guess they are sort of all over the place. 
Ill double check. thanks!

Reply:
They serve a specific purpose, i.e. so that the threaded code works nicely and you get a clean error if the assertion was not met. This helper is not needed for the test at all, I think.

Reply:
I guess I can resolve it since I deleted this code?